app-arch/createrepo_c/createrepo_c-0.20.1-r1.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 2020-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="C implementation of createrepo"
+HOMEPAGE="https://github.com/rpm-software-management/createrepo_c"
+if [[ ${PV} = 9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/rpm-software-management/createrepo_c.git"
+else
+	SRC_URI="https://github.com/rpm-software-management/createrepo_c/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+
+IUSE="doc legacy test"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	app-arch/bzip2:=
+	app-arch/drpm
+	app-arch/rpm
+	app-arch/xz-utils
+	app-arch/zchunk
+	dev-db/sqlite:3
+	dev-libs/glib:2
+	dev-libs/libxml2
+	dev-libs/openssl:=
+	net-misc/curl
+	sys-apps/file
+	sys-libs/libmodulemd
+	sys-libs/zlib:=
+"
+
+RDEPEND="${DEPEND}"
```

Comment:
Dropping lzma?

Reply:
`app-arch/xz-utils` is the upstream dependency and provides liblzma/lzma.h